docs/getting-started/install.md

```diff
@@ -178,6 +178,34 @@ make check
 sudo make install
 ```
 
+#### Installing BLST
+
+Download and install BLST so that cardano-base can pick it up (assumes pkg-config is installed):
+
+```
+git clone https://github.com/supranational/blst
+cd blst
+git checkout v0.3.10
+./build.sh
+cat > libblst.pc << EOF
+prefix=/usr/local
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
+
+Name: libblst
+Description: Multilingual BLS12-381 signature library
+URL: https://github.com/supranational/blst
+Version: 0.3.10
+Cflags: -I${includedir}
+Libs: -L${libdir} -lblst
+EOF
```

Comment:
🙏 I hope this doesn't bitrot over time 😬

Reply:
I am sure it will, unfortunately. I have sent an upstream PR but not sure if or when it will be merged